js/src/components/ParagraphsWithQuiz.js

```diff
@@ -186,6 +188,15 @@ class ParagraphsWithQuiz extends React.Component {
               isQuiz
             />
           )}
+          <SnackAlert
+            show={this.state.alertOpen}
+            message={Drupal.t('Quiz submission failed. Please try again.', {}, { context: 'ANU LMS' })}
+            onClose={() => this.setState({ alertOpen: false })}
+            severity='warning'
```

Comment:
@gease why did you decide to use warning severity here?

Reply:
> why did you decide to use warning severity here?
Arbitrary choice. Change to whatever you like.

As for QuizOptionsHandler, this component seems to never be rendered. At least i failed to find where it is.

@kalabro ^^